site/content/en/contributions/design/metadata.md

```diff
@@ -0,0 +1,77 @@
+---
+title: "Metadata in XDS resources"
+---
+
+## Overview
+
+In Envoy, [static metadata][] can be configured on various resources: [listener][], [virtual host][], [route][] and [cluster][].
+
+Static metadata can be used for various purposes:
+- Observability: enrichment of access logs and traces with [metadata formatters][] and [custom tags][]. 
+- Processing: provide configuration context to filters in a certain scope (e.g. vhost, route, etc.).  
+
+This document describes how Envoy Gateway manages [static metadata][] for various XDS resource such as listeners, virtual hosts, routes, clusters and endpoints.
+
+## Configuration 
+
+Envoy Gateway propagates certain attributes of [Gateway-API][gw_api] resources to XDS resources. Attributes include:
+- Metadata: Kind, Group/Version, Name, Namespace and Annotations (belonging to the `metadata.gateway.envoyproxy.io` namespace)
+- Spec: SectionName (Listener Name, RouteRule Name, Port Name), in-spec annotations (e.g. Gateway Annotations)
+
+Future enhancements may include:
+- Additional attribute propagation
+- Supporting section-specific metadata, e.g. HTTPRoute Metadata annotations that are propagated only to a specific route rule XDS metadata.
+- Supporting additional XDS resource, e.g. endpoints and filter chains.
+
+## Translation 
+
+Envoy Gateway uses the following namespace for envoy resource metadata: `io.envoyproxy.gateway.metadata`. For example, an envoy [route][] resource may have the following metadata structure:
+
+```yaml
+name: httproute/gateway-conformance-infra/myroute/rule/0/match/0/*
+match:
+  path_separated_prefix: "/mypath"
+route:
+  cluster: httproute/gateway-conformance-infra/myroute/rule/0
+metadata:
+  filter_metadata:
+    io.envoyproxy.gateway.metadata:
+      namespace: gateway-conformance-infra
+      groupVersion: gateway.networking.k8s.io/v1
+      kind: HTTPRoute
+      annotations: 
+        foo: bar
+      name: myroute
+```
+
+Envoy Gateway translates [Gateway-API][gw_api] in the following manner:
+- [Gateway][gw] metadata is propagated to envoy [listener][] metadata. If merge-gateways is enabled, [Gateway Class][gc] is used instead. 
+- [Gateway][gw] metadata and Listener Section name are propagated to envoy [virtual host][] metadata
+- [HTTPRoute][httpr] and [GRPCRoute][grpcr] metadata is propagated to envoy [route][] metadata. When Gateway-API adds support [named route rules][], the route rule name
+- [TCP/UDPRoute][tcpr] and [TLSRoute][tlsr] resource attributes are not propagated. These resources are translated to envoy filter chains, which do not currently support static metadata. 
+- [Service][svc], [ServiceImport][svci] and [Backend][] metadata and port name are propagated to envoy [cluster] metadata.
+
+## Usage
```

Comment:
about `usgae`, can we also highlight this in the https://gateway.envoyproxy.io/latest/tasks/observability/proxy-observability/#logs ?